spec/controllers/members_controller_spec.rb

```diff
@@ -38,4 +38,48 @@
       end
     end
   end
+
+  describe '.create' do
+    before do
+      ENV['SMTP_USER_NAME'] = 'username'
+      @group = Group.create!(name: 'Design')
+      @user = Member.create!(name: 'John Doe', email: 'jd@artsymail.com', group_ids: [@group.id])
+      @user = Member.find_by_email('jd@artsymail.com')
```

Comment:
Assigns `@user` twice? Also this is a member, not a user, assign to `@member`. Even better, use RSpec's `let`, so

```ruby
let(:member) { Member.create! ...}
```

Even better use the fabricator gem.

```ruby
let(:member) { Fabricate(:member) }
```